stable_gimpfusion.py

```diff
@@ -364,6 +364,8 @@ def getControlNetParams(self, cn_layer):
             # ControlNet image size need to be in multiples of 64
             layer64 = layer.copy().insert().resizeToMultipleOf(64)
             data.update({"input_image": layer64.toBase64()})
+            global settings
+            data.update({"models": settings.get("model")})
```

Comment:
Should this be `model` instead of `models` according to the Examples in the [ControlNet Wiki](https://github.com/Mikubill/sd-webui-controlnet/wiki/API#examples-1)

Reply:
Yes, according to documentation that is correct, however this version using `models` seems to be working. Using `model` throws a runtime error from the [controlney.py](https://github.com/Mikubill/sd-webui-controlnet/blob/main/scripts/controlnet.py#L338-L339) source library.

Reply:
This modification was made in error as the `layer.loadData` shouild have the `model` field filled.